From 72f6b36a48ddb6a109ad2f665402844e63fffa73 Mon Sep 17 00:00:00 2001
From: Lili Li <lili.li@intel.com>
Date: Wed, 30 Jun 2021 13:52:54 +0800
Subject: [PATCH 2/2] drm: hantro: drop the _rcu postfix on function names

API changed since commit c1a050546c12 ("dma-buf: drop the _rcu postfix on function names v3")

Signed-off-by: Lili Li <lili.li@intel.com>
---
 drivers/gpu/drm/hantro_driver/hantro_fence.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/gpu/drm/hantro_driver/hantro_fence.c b/drivers/gpu/drm/hantro_driver/hantro_fence.c
index f58f0dc23233..6e621e998063 100644
--- a/drivers/gpu/drm/hantro_driver/hantro_fence.c
+++ b/drivers/gpu/drm/hantro_driver/hantro_fence.c
@@ -174,7 +174,7 @@ int hantro_acquirebuf(struct drm_device *dev, void *data,
 	}
 
 	/* Check for a stalled fence */
-	if (!dma_resv_wait_timeout_rcu(resv, arg->flags & HANTRO_FENCE_WRITE, 1,
+	if (!dma_resv_wait_timeout(resv, arg->flags & HANTRO_FENCE_WRITE, 1,
 				       timeout)) {
 		ret = -EBUSY;
 		goto err;
@@ -262,7 +262,7 @@ int hantro_testbufvalid(struct drm_device *dev, void *data,
 	}
 
 	/* Check for a stalled fence */
-	if (dma_resv_wait_timeout_rcu(resv, 1, 1, 0) <= 0)
+	if (dma_resv_wait_timeout(resv, 1, 1, 0) <= 0)
 		arg->ready = 0;
 	else
 		arg->ready = 1;
-- 
2.27.0

